### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by scopatz**
This adds some simple sphinx docs to the repo. CC @marcelotrevisani

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
Docs are always welcome and helpful! :tada: 
haha

I think that is a nice addition, we just need to understand and fix why it is failing on ``osx``

**Comment by scopatz** [comment]
I think the docs are failing due to changes on azure: https://gitter.im/conda/conda?at=5dbda6059825bd6bac13d5c2

Maybe we should just skip these for now

File: docs/dependencies.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Conda-press currently has the following external dependencies,
+
+*Run Time:*
+
+#. xonsh
+#. lazyasd
+#. ruamel.yaml
+#. tqdm
+#. requests
+#. conda (installed in the current environment)
```
Comment (thewtex) [inline docs/dependencies.rst]: "All of these dependencies are available in `conda-forge`."

?

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+**Issue 2:**
+
+Many packages with compiled extensions do not have wheels available on one or more
+popular platforms (Windows, Mac, Linux). This is because building wheels can
+be very difficult.  Conda has a lot of packages that are not available as wheels otherwise.
+Conda-press allows these packages to easily become generally usable wheels.
+
+**Issue 3:** Some people want a package index built on newer ABIs than `manylinux<N>`
+
+
```
Comment (thewtex) [inline docs/index.rst]: Does conda-press also address the issue of shipping and making shared library dependencies loadable at runtime?

**Comment by marcelotrevisani** [comment]
> I think the docs are failing due to changes on azure: https://gitter.im/conda/conda?at=5dbda6059825bd6bac13d5c2
> 
> Maybe we should just skip these for now

Yes, indeed, we might skip it for now
I will take a look a bit further on that with #48

**Comment by marcelotrevisani** [comment]
I think if we merge the https://github.com/regro/conda-press/pull/48 we can rebase it and integrate this PR
Or we just integrate it, as it is just docs there is no reason to run the tests :)

File: docs/index.rst (hunk 2 above)
Comment (marcelotrevisani) [reply docs/index.rst]: Usually, it works well. But I think there are some corner cases which we may not being covered.

@scopatz might explain it better :)

File: docs/dependencies.rst (hunk 1 above)
Comment (marcelotrevisani) [reply docs/dependencies.rst]: Sorry, but what do you mean? Do you think would be helpful to add that sentence or you are actually asking? But all those dependencies are in ``conda-forge`` indeed.

File: docs/index.rst (hunk 2 above)
Comment (scopatz) [reply docs/index.rst]: Added some docs to this point.

**Comment by scopatz** [comment]
Merging this so I can build docs for tomorrow
